src/ActionInterface.cpp

```diff
@@ -369,3 +369,19 @@ ActionInterface::ExchangeRadioFrequencies(bool to_devices) noexcept
     ActionInterface::SetStandbyFrequency(old_active_freq, old_active_freq_name, to_devices);
   }
 }
+
+void
+ActionInterface::SetTransponderCode(TransponderCode code, bool to_devices) noexcept
+{
+  /* update interface settings */
```

Comment:
Please add `assert(code.IsDefined())` at the start of the function so bugs passing malformed values can be detected when debug builds are used.

Reply:
Agree!